development/scripts/README.md

```diff
@@ -0,0 +1,13 @@
+# Scripts
+
+## `generate-data-files.py`
```

Comment:
these files also appear in the other PR -- should one of the PRs be merged first and then the other one or is there duplicate content?